### PR TITLE
#85 add codestyle max line length

### PR DESCRIPTION
**Pull request by PiaSophieBe**

### CODE REVIEW

**Comment by mzudse** [comment]
@PiaSophieBe You could set the severity to warning, because of the 63 errors checkstyle will fail the build.

A new issue could be created to fix the errors and once that's done, we set the severity back to error.

`<property name="severity" value="warning"/>`

File: src/main/style/checkstyle.xml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
-          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
 
 <module name="Checker">
 
     <property name="charset" value="UTF-8" />
 
+    <property name="severity" value="warning"/>
```
Comment (astanik) [inline src/main/style/checkstyle.xml:10]: Why is the severity reduced to warning?

**Comment by astanik** [comment]
@PiaSophieBe It would be good if we could complete this pull request. There are also further developments underway that could potentially lead to merge conflicts again.

**Review by astanik — APPROVED** [review]
Please create an issue to change the severity back to Error.

**Pull request merged**